scripts/environment.sh

```diff
@@ -24,7 +24,7 @@ function os_check() {
     export DISTRO="${ID}${VERSION_ID%.*}"
     export OS="${ID}"
     export OS_VERSION_ID=$VERSION_ID
-    export SUPPORTED_DISTROS=(centos8 rhel8 ubuntu20)
+    export SUPPORTED_DISTROS=(centos8 centos9 rhel8 ubuntu20)
 
     if [[ ! "${SUPPORTED_DISTROS[*]}" =~ $DISTRO ]]; then
         echo "Supported OS distros for the host are: CentOS Stream 8 or RHEL8 or Ubuntu20.04"
```

Comment:
```suggestion
        echo "Supported OS distros for the host are: CentOS Stream 8/9 or RHEL8/9 or Ubuntu20.04"
```

Reply:
Aha, true, I have to run the tests again though, I will run the integration test only when I change this and once e2e passes on this PR. By the way why are you suggesting RHEL 9 also, does it mean supporting CentOS 9 also supports RHEL 9?

Reply:
Once e2e passes and pushing, we can just re-run integration tests, agree

Reply:
Ok, I will change it to add RHEL 9 as well